src/django_components/templatetags/component_tags.py

```diff
@@ -480,13 +481,27 @@ def mark_kwarg_key(key: str, is_agg_key: bool) -> None:
 
     # Post-process args/kwargs - Mark special cases like aggregate dicts
     # or dynamic expressions
+    args: List[Expression] = []
+    for val in raw_args:
+        if is_dynamic_expression(val.token):
+            args.append(DynamicFilterExpression(parser, val.token))
+        else:
+            args.append(val)
+
     kwarg_pairs: RuntimeKwargPairsInput = []
     for key, val in raw_kwarg_pairs:
         is_spread_op = is_internal_spread_operator(key + "=")
 
         if is_spread_op:
-            expr = parser.compile_filter(val.token)
+            # Allow to use dynamic expressions with spread operator, e.g.
+            # `..."{{ }}"`
```

Comment:
This is just crazy, I not sure I think we should include this. Will confuse everyone that sees and expression like this!